### DISCUSSION TITLE
How can I trigger Tooltip in mobile

### DISCUSSION THREAD

**Discussion by myncflaconi** [radix-ui/primitives#2866] [Help]
Hello there,
I am integrating tooltip with my design system library. Normally, I would like to use the my component in mobile devices too.
I couldn't find a way to trigger tooltip in mobile view because it is designed to get triggered only on hover. 

Is there a way to trigger it on tap by passing some props. My first choice is not to add any hack. 
Try this link on your mobile device https://www.radix-ui.com/primitives/docs/components/tooltip#root

Thanks in advance

**Comment by davidduncandev** [comment]
Tooltips in the WAI-ARIA sense (role="tooltip, opened on hover and focus) won't work well on touch devices. I've hit the same issue recently and tried to ensure designs are created without tooltips as far as possible. The toggletip pattern is maybe worth looking into since that operates on click (https://inclusive-components.design/tooltips-toggletips/) but I'm not sure if your use cases fit the toggletip pattern or not. A popover might also work. The popover would transfer focus to announce the bubble content, whereas the toggletip is a simpler api which would use live regions.

**Comment by 24-20** [comment]
wondering the same thing right now, did any of you guys find a solution for this?
